### PR TITLE
interactive list

### PR DESCRIPTION
**Pull request by hopeforkoami**
in this branch I,
-Add a new methods related to the status updates (completed: true / false).
-Add event listener to the checkbox (change).
-Update items object's value for completed key upon user actions.
-Implement a function for the "Clear all completed"
-Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -43,11 +43,53 @@ function displayComponent(elmnts) {
...
+    elmnts.forEach((elmnt) => {
+      const test = elmnt.classList;
+      if (test.contains('line_through')) {
+        test.remove('line_through');
+      }
+    });
+  } else {
+    const test = elmnts.classList;
+    if (test.contains('line_through')) {
+      test.remove('line_through');
```
Comment (tufoinnkuo10) [inline src/index.js:67]: - [ ] Great job so far.. please ensure that your todos are stored in the local storage, when i refresh the page everything disappears. please fix that
- [ ] kindly ensure that when the clear complete button is clicked, all items that have been checked should be removed and not just one
- [ ] please ensure that immediately an item is checked, its completed status changes to true.

Screenshot (491)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @hopeforkoami 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionallyl

Required Changes ♻️
Check the comments under the review
Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/index.js [hunk 2]
```diff
@@ -43,11 +43,55 @@ function displayComponent(elmnts) {
...
+  displayComponent(document.querySelector(`.details${index}`));
+  removeLineThroughComponent(document.querySelector(`.input${index}`));
+  tasklist.localData[index - 1].setCompleted(false);
+  tasklist.refreshIndex();
+}
+
+function clearCompletedTasks() {
+  tasklist.clearCompletedTasks();
+  refreshTodoList();
+}
```
Comment (Kamba56) [inline src/index.js:94]: - [Optional] Consider adding your functions in separate modules and import them to your index.js file. This is best ES6 practice.

**Review by Kamba56 — APPROVED** [review]
Hi @hopeforkoami, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏
!👏👏👏
!👏👏👏
!👏👏👏
!👏👏👏
!👏👏👏
!👏👏👏
!👏👏👏

## Highlights
- [x] Great use of Github flow
- [x] All linters passed

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**